TESTS/netsocket/README.md

```diff
@@ -348,156 +423,144 @@ mbed test -n mbed-os-tests-network-*,mbed-os-tests-netsocket*
 description.
 
 ### SOCKET_OPEN_DESTRUCT
 
 **Description:**
 
-Call `Socket::open()` and then destruct the socket.
+Call `Socket::open()`, and then destruct the socket.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create an object by calling `new Socket()`.
-2.  Call `Socket::open(stack)`.
-3.  Delete the object.
-4.  Repeat 100 times.
+1. Create an object by calling `new Socket()`.
+1. Call `Socket::open(stack)`.
+1. Delete the object.
+1. Repeat 100 times.
 
 **Expected result:**
 
-`Socket::open()` should always return `NSAPI_ERROR_OK`.
-
-
+`Socket::open()` always returns `NSAPI_ERROR_OK`.
 
 ### SOCKET_OPEN_LIMIT
 
 **Description:**
 
- Call `Socket::open()` until it runs out of memory or other internal limit
-in the stack is reached.
+ Call `Socket::open()` until it runs out of memory or another internal limit in the stack is reached.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create a object by calling `new Socket()`.
-2.  Call `Socket::open(stack)`.
-3.  Repeat until `NSAPI_ERROR_NO_MEMORY` or `NSAPI_ERROR_NO_SOCKET` error code is returned.
-4.  Call "delete" for all previously allocated sockets.
-5.  Repeat.
+1. Create an object by calling `new Socket()`.
+1. Call `Socket::open(stack)`.
+1. Repeat until `NSAPI_ERROR_NO_MEMORY` or `NSAPI_ERROR_NO_SOCKET` error code is returned.
+1. Call "delete" for all previously allocated sockets.
+1. Repeat.
 
 **Expected result:**
 
-Should be able to reserve at least 4 sockets. After freeing all sockets, should  be able to reserve same number of sockets.
-
-
+It can reserve at least four sockets. After freeing all sockets, it can reserve the same number of sockets.
 
 ### SOCKET_OPEN_TWICE
 
 **Description:**
 
-Call `Socket::open()` twice
+Call `Socket::open()` twice.
 
 **Preconditions:**
 
-1.  Network interface and stack are initialized.
-2.  Network connection is up.
+1. Network interface and stack are initialized.
+1. Network connection is up.
 
 **Test steps:**
 
-1.  Create a object by calling `new Socket()`.
-2.  Call `Socket::open(stack)`.
-3.  Call `Socket::open(stack)`.
-4.  Delete the socket.
+1. Create an object by calling `new Socket()`.
+1. Call `Socket::open(stack)`.
+1. Call `Socket::open(stack)`.
+1. Destroy the socket.
 
 **Expected result:**
 
-`Socket::open()` should first return `NSAPI_ERROR_OK` and second call `NSAPI_ERROR_PARAMETER`.
-
-
+`Socket::open()` first returns `NSAPI_ERROR_OK` and then calls `NSAPI_ERROR_PARAMETER`.
```

Comment:
This is now incorrect. Both `NSAPI_ERROR_OK` and `NSAPI_ERROR_PARAMETER` are return values that `Socket::open()` can return.

When `Socket::open()` is first called, it returns `NSAPI_ERROR_OK`, when `Socket::open()` is called second time, it returns `NSAPI_ERROR_PARAMETER`.

Reply:
Fix that by change to more datailed description:
first call (socket was not opened) returns `NSAPI_ERROR_OK` and nexts calls (socet was opened by previous call Socket::open()) return `NSAPI_ERROR_PARAMETER`.